app/mailers/admin_mailer.rb

```diff
@@ -3,10 +3,14 @@
 # Admin Mailer on Software Record request
 class AdminMailer < ApplicationMailer
   default from: 'uclappdev@uc.edu'
-  def send_mail(_id, _name)
-    mail(
-      to: 'mallepvl@mail.uc.edu',
-      subject: 'New Software Request'
-    )
+
+  def new_software_request_mail(id, name)
+    @id = id
+    @name = name
+    @template = 'new_software_request_mail.html.erb'
+    @users = User.where(roles: 'root_admin')
+    @users.each do |admin|
```

Comment:
@Uzval-Mallepeddi The mail command will never run if there are no admin users.  That's why your admin_mailer_spec is failing.  CircleCI starts with a blank test database with no admin users.  So you need to create an admin user in your spec and it should pass.

Reply:
Alright! Thanks Glen. That worked like a charm!